docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -271,3 +271,108 @@ FETCH FIRST N ROWS``.
 
 Implementation and support is connector-specific since different data sources
 support different SQL syntax and processing.
+
+We will be showcased two similar queries for demonstrating how the Trino
+compute engine deals with a connector supporting Top-N functionality and also with a
+connector which is missing this functionality.
+
+Following is  presented a concrete example of a Top-N query
+on the ``tpch`` connector which does not support Top-N pushdown functionality::
+
+    SELECT custkey, name
+    FROM tpch.sf1.customer
+    ORDER BY custkey
+    LIMIT 5;
+
+You can get the explain plan by prepending the above query with ``EXPLAIN``::
+
+    EXPLAIN SELECT custkey, name
+    FROM tpch.sf1.customer
+    ORDER BY custkey
+    LIMIT 5;
+
+Following is presented the query plan of the ``EXPLAIN`` query.
```

Comment:
The plan may change between versions so maybe add a note that the exact plan may look different?

Alternatively only show the relevant parts of the plan instead of entire plan?

Reply:
I'd argue that the plan is relatively small and rather easy to comprehend.
I'm reluctant in removing parts of the plan because also the other pushdown sections are presenting the entire plan and also the user may benefit of having all the plan details at hand to better understand how the query is being handled by Trino.

Reply:
I would leave it ... and if the plan changes .. thats a bug in the documentation and we just recreate it

Reply:
Fair enough.